clustered/integration-test/src/test/java/org/ehcache/clustered/replication/DuplicateTest.java

```diff
@@ -148,7 +147,6 @@ private StoreAccessException findStoreAccessException(Object[] objects) {
         return (StoreAccessException) o;
       }
     }
-    fail("There should be an exception somewhere in " + Arrays.toString(objects));
-    return null;
+    throw new AssertionError("There should be an exception somewhere in " + Arrays.toString(objects));
```

Comment:
Any reason you didn't use `org.junit.Assert#fail(java.lang.String)`?

Reply:
Same reason as above.